server/sonar-web/src/main/js/helpers/README.md

```diff
@@ -0,0 +1,90 @@
+## Making Requests
+
+Since 6.0 all API requests must provide CSRF protection token. In order to
+help you to do so, we provide a set of useful helpers.
+
+### Getting Started
+Let's start with a simple GET request.
+
+```
+window.sonarqube.appStarted.then(function () {
+  // some code here
+  window.SonarRequest
+      .getJSON(window.baseUrl + '/api/issues/search')
+      .then(function (response) {
+        // here 'response' contains the object representing the JSON output
+      });
+});
+```
+
+`window.sonarqube.appStarted` returns a `Promise` telling you when the
+application is stated and all the runtime libraries and hooks are installed.
+
+`window.SonarRequest` contains all the helper methods to do an API requests.
+
+`window.SonarRequest.getJSON` is a simplest helper to do an API call,
+receive some data and parse it as JSON.
+
+### API Documentation
+
+#### `window.sonarqube.appStarted: Promise`
+A promise telling when the application has started.
+You must put your code in the `resolve` path.
+
+#### `window.SonarRequest.request(url: string): Request`
+Start making an API call. `url` must start from `window.baseUrl`.
```

Comment:
@simonbrandhof it's definitely a good idea. However I don't know how to do this without breaking compatibility with the governance plugin.